plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1808,6 +1808,8 @@ class account_failures {
 public:
    constexpr static uint32_t max_failures_per_account = 3;
 
+   explicit account_failures( const eosio::subjective_billing& sb ) : subjective_billing(sb) {}
```

Comment:
Add comment that lifetime of `sb` must outlive `account_failures`

Reply:
Addressed in ac9936c